Misc/NEWS.d/next/Library/2022-06-06-12-58-27.gh-issue-79579.e8rB-M.rst

```diff
@@ -0,0 +1,2 @@
+:mod:`sqlite3` now correctly detects DML queries, even when prefixed with
+comments. Patch by Erlend E. Aasland.
```

Comment:
I think the NEWS entry can still be improved. Perhaps:

```suggestion
:mod:`sqlite3` now strips leading whitespace and comments from SQL queries,
thus improving DML detection. Patch by Erlend E. Aasland.
```

Reply:
The current code already skips whitespace.
Something like this: Correctly detect DML statement when there are leading comments.

Reply:
Thanks!